### PR TITLE
fix getreserves value display

### LINKED ISSUE #5035: display values for getReserves() for token pairs
Token pairs should display the following three values when [viewing token pairs](https://blockscout.com/xdai/mainnet/token/0x4505b262DC053998C10685DC5F9098af8AE5C8ad/read-contract):

Example:
```
_reserve0: 228152537528654640237589
_reserve1: 1026156125
_blockTimestampLast: 1639659339
```

### PR DESCRIPTION
**Pull request by zalam003**
Closes #5035 

### Enhancements
*Display getReserves values*

## Checklist for your Pull Request (PR)

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

**Review by vbaranov — COMMENTED** [review]
@zalam003 please attach the screenshots of the visual changes: before/after changes.

**Comment by zalam003** [comment]
@vbaranov We already updated our deployment.  But you can see the difference below.

Here is the [before](https://blockscout.com/xdai/mainnet/address/0x0E3e9CCeb13c9f8c6FAF7a0f00F872d6291630dE/read-contract):

image



and [after](https://explorer.energi.network/address/0x5Cc8B3D89AE2cA30A08b6697E33606FC3437d392/read-contract):

image

File: apps/block_scout_web/lib/block_scout_web/views/smart_contract_view.ex [hunk 1]
```diff
@@ -326,11 +326,11 @@ defmodule BlockScoutWeb.SmartContractView do
   end
 
   defp render_type_value(type, value, type) do
-    "<div style=\"padding-left: 20px\">(#{type}) : #{value}</div>"
+    "<div class=\"py-2 pr-2 text-nowrap\" style=\"padding-left: 20px\">(#{type}) : #{value}</div>"
```
Comment (nikitosing) [inline apps/block_scout_web/lib/block_scout_web/views/smart_contract_view.ex:329]: These changes also affect on the other elements:
Before: 

image

After:

image

I think better to return old view (namely indentation and black color for the names) for the such function response boxes
contract on the screenshot:
https://blockscout.com/poa/sokol/address/0xB9f191f5c324B4c97B4c11333338288918c5968C/read-contract
method: `getReceipt`
input data: `0x9cd14677f9aa5569b3bbb351fcd67d1115aa563ae1624276d3d4e67fb3d6f926`

**Review by nikitosing — CHANGES_REQUESTED** [review]

**Comment by nikitosing** [comment]
Also, please rebase on master branch and run the following commands to fix gettext error:
`cd apps/block_scout_web`
`mix gettext.extract --merge`

**Comment by zalam003** [comment]
@nikitosing We have customized our implementation at Energi.  We are not ready to open source it yet.  However, we want to share some of the changes with you as and when we feel it may benefit you.  We felt this is something Blockscout should include.  We leave it up to you if you want to merge or reject. Or refactor to your specifications if needed.

We have implemented it to match what you see for LP tokens in [etherscan.io](https://etherscan.io/token/0xb6916bc20cae34de64af39b8534d1459d8bb4128#readContract).  Following is a screenshot from Etherscan.

image

**Comment by vbaranov** [comment]
Closed in favor https://github.com/blockscout/blockscout/pull/5197

**Pull request closed without merging**